src/stylesheets/base/_colors.scss

```diff
@@ -0,0 +1,14 @@
+$colors: (
+  cool-charcoal:        #252224,
+  warm-charcoal:        #504949,
+  deep-blue:            #184D66,
+  emerald:              #0E5254,
+  teal:                 #1793AD,
+  mint:                 #D3E7E2,
+  grey:                 #E2EBE9,
+  white:                #FFFFFF,
+  yellow:               #F2C84D,
+  orange:               #C06D3A,
+  coral:                #EA4446,
+  gradient-background:  linear-gradient(#184D66 3.08%, #D3E7E2 94.99%)
```

Comment:
Were the percentages from Figma? Once you get to applying this gradient, I'm curious to know if 0% and 100% would work just as well?

Reply:
These were in Figma and seem oddly specific? Will probably be fine do just go with your suggestion.